### PR TITLE
SHPDR-17 implemented set-style command and small fixes

### PR DESCRIPTION
**Pull request by fokija**

### CODE REVIEW

File: command.cc [hunk 1]
```diff
@@ -182,10 +222,11 @@ StringArgs parseUserInput(const std::string& fullCmdStr)
     std::string token = "";
     size_t inputSize = fullCmdStr.size();
 
-    for (auto i=0u; i < inputSize; i++)
+    for (auto i = 0u; i < inputSize; i++)
     {
         char c = fullCmdStr[i];
-        if (std::isalnum(c) != 0 || c == '-')
+        std::string availableSpecialChar = "!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~";
```
Comment (pietrik123) [inline command.cc:228]: what is it for ? do we need all of these for any command ?

File: command.cc [hunk 2]
```diff
@@ -133,6 +132,43 @@ bool DrawCmd::execute()
...
 
+SetStyleCmd::SetStyleCmd(CommandReceiver&r, const StringArgs& args) : Command(r, args)
+{
+}
+
+bool SetStyleCmd::execute()
+{
+    Shapes& shapesVector = m_receiver.getShapesVector();
+    std::cout << "\n";   
+    std::string x = m_args[1];
```
Comment (pietrik123) [inline command.cc:143]: please replace `x` with more decriptive variable name ?

File: command.cc [hunk 3]
```diff
@@ -133,6 +132,43 @@ bool DrawCmd::execute()
...
+    if (m_args[0] == "all")
+    {
+        for (size_t i = 0; i < shapesVector.size(); i++)
+        {
+            shapesVector[i]->setChar(tempChar);
+        }
+    }
+    else 
+    {
+        int y = std::atoi(m_args[0].c_str());
```
Comment (pietrik123) [inline command.cc:158]: please see a comment for `x` variable

File: command.cc [hunk 4]
```diff
@@ -133,6 +132,43 @@ bool DrawCmd::execute()
...
+SetStyleCmd::SetStyleCmd(CommandReceiver&r, const StringArgs& args) : Command(r, args)
+{
+}
+
+bool SetStyleCmd::execute()
+{
+    Shapes& shapesVector = m_receiver.getShapesVector();
+    std::cout << "\n";   
+    std::string x = m_args[1];
+    char tempChar = x[0];
```
Comment (pietrik123) [inline command.cc:144]: let's return `false` if the second parameter's length is more than 1 and inform the user that single character should be provided

**Review by pietrik123 — CHANGES_REQUESTED** [review]

File: command.cc (hunk 1 above)
Comment (pietrik123) [reply command.cc:228]: after we discussed, we should keep your change or we may use 
```
if (c > 32 && c < 127)
{
}
```

**Review by pietrik123 — APPROVED** [review]

**Review by dante7867 — APPROVED** [review]

**Pull request merged**